app/serializers/book_serializer.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class BookSerializer < ActiveModel::Serializer
```

Comment:
So, you recommend to put al the serializers in the generic folder ```serializers``` , then I'll have to rename the index and show serializers to something like ```book_show_serializer``` and ```book_index_serializer```?

I find using separate folders for each model more organized, maybe if I move the default ```book_serializer``` to this folder will be better?

Reply:
I preffer that, yes.
Also, I think its a good practice, at least in my opinion, using inheritance in serializers, because sometimes you will return the same fields and by doing that, you only need to add only in the "Father"